xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsTranslations.xml

```diff
@@ -51,7 +51,8 @@ XWiki.Mentions.MentionsConfigurationClass_mentionsColor=Mentions Color
 XWiki.Mentions.MentionsConfigurationClass_mentionsColor.hint=The color of the mentions to the current user (note: used as the value of a background-color css attribute).
 XWiki.Mentions.MentionsConfigurationClass_selfMentionsColor=Self Mentions Color
 XWiki.Mentions.MentionsConfigurationClass_quoteActivated=Activate the quotes
-XWiki.Mentions.MentionsConfigurationClass_quoteActivated.hint=Defines if a quote of the text surrounding a mention is included in the user mention notifications.</content>
+XWiki.Mentions.MentionsConfigurationClass_quoteActivated.hint=Defines if a quote of the text surrounding a mention is included in the user mention notifications.
+mentions.tip=Mention a user by typing @ then the user name in the Wysiwyg editor.</content>
```

Comment:
Not sure if I would mention the Wysiwyg editor: I'm a bit afraid that users will associate it only with "edit Wysiwyg" while it now works also in the in-place editor and in the comment editor. At the very least, I would change it to "a wysiwyg editor", to clarify a bit that it could be any wysiwyg editor.

Reply:
Translation updated for en and fr as well.

Reply:
The text is a bit strange, at least in EN. I would change it to:

"When in the WYSIWYG editor, you can mention a user by typing the '@' character followed by the user's name (you can type some letters and you'll get autocompletion)."

@surli I don't like "a wysiwyg editor", it's really weird for users. From a user POV there's a single WYSIWYG editor. It's too weird to say "a wysywig editor", it creates a cognitive dissonance.

Reply:
> @surli I don't like "a wysiwyg editor"

not a big fan either, but I don't see how to express the fact that you could use it basically in any "rich text editor". Not only *the* WYSIWYG editor. Else we can explicitely say that they can use it on comment editor too at least.

Reply:
Just consider that for users the WYSIWYG editor is CKEditor (ie the editor with the toolbar, etc). Whether it's inplace or not, I don't think it matters.

Reply:
WYSIYWG means WYSIWYG. It's not a specific editor. It's a generic concept.

Reply:
Otherwise you say the "Inplace editor" or the "Standard editor" (it doesn't have a name right now which is a problem BTW).

Reply:
Inplace editor is not much a problem, as I mention it's mainly about the comment editor that we should explicit.

Reply:
I've update the translation with your proposal, and a translation in french.

Reply:
> Inplace editor is not much a problem, as I mention it's mainly about the comment editor that we should explicit.

comments editor is also WYSIWYG now.

Reply:
> It's too weird to say "a wysywig editor", it creates a cognitive dissonance.

Coming back on that since @manuelleduc translated it "un éditeur WYSIWYG" and not "l'éditeur WYSIWYG". That cognitive dissonance which you don't like is actually wanted. It says to the user "hey, I'm not talking specifically about the WYSIWYG editor you use to see when clicking edit, but all the WYSIWYG editors available in xwiki, so try that also in comments, or in objects!"
Really I'm pretty sure that @manuelleduc translation will be fixed in no-time with "l'éditeur WYSIWYG" (which is right) and that all users will only think "ok so I cannot do it in comments".

Reply:
> Really I'm pretty sure that @manuelleduc translation will be fixed in no-time with "l'éditeur WYSIWYG" (which is right) and that all users will only think "ok so I cannot do it in comments".

We'll see but for me if you use the WYSIWYG editor for comments then it matches "l'éditeur WYSIWYG".